public/documentation/about/faq.md

```diff
@@ -93,6 +93,18 @@ Also note that we don't have a way to package your app with Crosswalk for deploy
 
 Crosswalk does not support iOS at this time.
 
+### <a id="How-to-proguard-Crosswalk"></a>How to proguard Crosswalk?
+
+Don't proguard xwalk library because of broken JNI link. Add following ProGuard rules in proguard-project.txt file.
+
+    -keep class org.xwalk.core.** {
+        *;
+    }
+    -keep class org.chromium.** {
+	    *;
+    }
+    -keepattributes **
+
```

Comment:
Does -keepattributes have two ** or one * ?  In the original bug, Rory Kelly put:

-keep class org.xwalk.** { *; }
-keep class org.chromium.** { *; }
-keepattributes *   

I know nothing about ProGuard, just making sure we have tried these values successfully.